test/commons/text/is-human-interpretable.js

```diff
@@ -0,0 +1,55 @@
+describe('text.isHumanInterpretable', function() {
+	it('returns 0 when no given string is empty', function() {
+		var actual = axe.commons.text.isHumanInterpretable('');
+		assert.equal(actual, 0);
+	});
+
+	it('returns 0 when given a single character that is blacklisted as icon', function() {
```

Comment:
Either put the char in the test name, or do all of these assertions in the same test, but don't use the same test name for different tests.